osgar/drivers/gps.py

```diff
@@ -140,7 +179,8 @@ def process_gen(self, data):
             ret = self.process_packet(packet)
             if ret is not None:
                 for k, v in ret.items():
-                    yield k, v
```

Comment:
well, the receiver is maybe interested that data from GPS are coming but there is no signal?

Reply:
I think it follows the original code because there is an `assert`t for None in the following code.